corpus/events/forms.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from .models import Event
+from config.models import Society
+from accounts.models import ExecutiveMember
+
+# Create forms here
+
+class EventForm(forms.Form):
+
+    society = forms.ModelChoiceField(
+        queryset=Society.objects.all(),
+        widget=forms.Select(attrs={'class': 'select select-bordered w-full'}),
+        empty_label="Select Society"
+    )
+
+    poc = forms.ModelChoiceField(
+        queryset=ExecutiveMember.objects.all(),
+        widget=forms.Select(attrs={'class': 'select select-bordered w-full'}),
+        empty_label="Select POC"
+    )
+
+    meeting_url = forms.URLField(widget=forms.URLInput(attrs={'class': 'input input-bordered w-full'}), required=False)
+    instagram_url = forms.URLField(widget=forms.URLInput(attrs={'class': 'input input-bordered w-full'}), required=False)
+    linkedin_url = forms.URLField(widget=forms.URLInput(attrs={'class': 'input input-bordered w-full'}), required=False)
+    visibility = forms.CharField(widget=forms.TextInput(attrs={'class': 'input input-bordered w-full'}), required=False)
```

Comment:
should this be forms.choicefield or forms.charfield?

Reply:
If you have like a set of visibilities that you define, better to make it a choice field.